### PR TITLE
add more files to protected file list

### PR DESCRIPTION
**Pull request by panyx0718**
test=develop

### CODE REVIEW

File: paddle/scripts/paddle_build.sh [hunk 1]
```diff
@@ -487,7 +487,17 @@ function assert_api_spec_approvals() {
         BRANCH="develop"
     fi
 
-    API_FILES=("paddle/fluid/API.spec" "paddle/fluid/framework/operator.h")
+    API_FILES=("paddle/fluid/API.spec"
+               "paddle/fluid/framework/operator.h"
+               "paddle/fluid/framework/tensor.h"
+               "paddle/fluid/framework/lod_tensor.h"
+               "paddle/fluid/framework/selected_rows.h"
+               "paddle/fluid/framework/op_desc.h"
```
Comment (typhoonzero) [inline paddle/scripts/paddle_build.sh:495]: how about adding `framework.proto` to replace `xx_desc.h`?

Comment (typhoonzero) [reply paddle/scripts/paddle_build.sh:495]: also `send_recv.proto.in`

Comment (panyx0718) [reply paddle/scripts/paddle_build.sh:495]: Added 2 more files. And keep xx_desc.h since they are also very important.

**Review by typhoonzero — APPROVED** [review]

**Review by qingqing01 — APPROVED** [review]
LGTM.

**Pull request merged**